omod/src/main/webapp/resources/scripts/metadata/manageVisitTypes.js

```diff
@@ -0,0 +1,118 @@
+angular.module("manageVisitTypes", [ "visitTypeService", "ngDialog", "ui.router", "uicommons.filters", "uicommons.common.error"])
+
+    .config([ "$stateProvider", "$urlRouterProvider", function($stateProvider, $urlRouterProvider) {
+        $urlRouterProvider.otherwise("/list");
+
+        $stateProvider
+            .state('list', {
+                url: "/list",
+                templateUrl: "templates/list.page",
+                controller: "ManageVisitTypesController"
+            })
+            .state("edit", {
+                url: "/edit/:visitTypeUuid",
+                templateUrl: "templates/edit.page",
+                params: {
+                    visitTypeUuid: null,
+                },
+                resolve: {
+                    visitType: function($stateParams, VisitType) {
+                        if ($stateParams.visitTypeUuid) {
+                            return VisitType.get({ uuid: $stateParams.visitTypeUuid, v: "full" });
+                        }
+                        return {};
+                    }
+                },
+                controller: "EditVisitTypeController"
+            });
+    }])
+
+    .controller("ManageVisitTypesController", [ "$scope", "$state", "VisitType", "VisitTypeService", "ngDialog",
+        function($scope, $state, VisitType, VisitTypeService, ngDialog) {
+            function sortWithRetiredLast(list) {
+                return _.sortBy(list, "retired");
+            }
+
+            function loadVisitTypes() {
+                // TODO standard function for failure of REST call
```

Comment:
Did you just copy and paste code and left these TODOs around, you should possibly remove them?

Reply:
Sorry, I will remove those comments